jf_agent/main.py

```diff
@@ -68,9 +76,11 @@ def main():
 
     jira_config = config.get('jira', {})
     bb_config = config.get('bitbucket', {})
+    git_config = config.get('git', bb_config)
```

Comment:
Looks a little weird. Is it your thinking that we still want to support the `bitbucket` config for backwards compatibility? If so then maybe it'd be a bit clearer to single-line it as:
```
git_config = config.get('git', config.get('bitbucket', {}))
```

Reply:
Yeah, I was trying to be backwards-compatible.  I can one-line it if you think that's clearer.